virt_who/provision.py

```diff
@@ -705,6 +705,7 @@ def jenkins_job_init(self, register_type, register_config, ssh_host, ssh_guest):
     def jenkins_parameter(self, hypervisor_config, register_config):
         parameter = list()
         parameter.append('-d TRIGGER_TYPE={0}'.format(deploy.trigger.type))
+        parameter.append('-d PROVISION_INI={0}'.format(self.get_exported_param("PROVISION_INI")))
```

Comment:
PROVISION_INI will not be used for trigger-gating job

Reply:
OK, will update with next PR